examples/sqlalchemy/README.rst

```diff
@@ -2,14 +2,36 @@
 SQLAlchemy Example
 ==================
 
+.. note::
+
+    TODO: Update this example to work with recent (2024) versions of Python and SQLAlchemy.
+
 A simple example of how one might use SQLAlchemy as a backing store for a 
 Connexion based application.
 
-Running:
+Preparing
+---------
+
+Create a new virtual environment and install the required libraries
+with these commands:
 
 .. code-block:: bash
 
+    $ python -m venv my-venv
+    $ source my-venv/bin/activate
+    $ pip install 'connexion[flask,swagger-ui,uvicorn]>=3.1.0'
     $ pip install -r requirements.txt
```

Comment:
We should remove the flask and connexio dependencies from `requirements.txt` if we install them directly on the command line.

Reply:
Ok, dropped the connexion and Flask lines from requirements.txt